presto-client/src/main/java/com/facebook/presto/client/ClientSession.java

```diff
@@ -32,6 +32,7 @@
 {
     private final URI server;
     private final String user;
+    private final String password;
```

Comment:
Password is an authentication option similar to Kerberos. It shouldn't be part of `ClientSession`.